### ISSUE TITLE
`--real-path`

### ISSUE THREAD

**Issue by drahnr** [sharkdp/fd#268]
The option `--absolute-path` creates "messy" results when used with relative dirs. As such I would propose another option similar to the bash command `realpath`.

**Comment by sharkdp** [comment]
Thank you for the feedback. Could you please give a (reproducible) example and what you would expect?

**Comment by sharkdp** [comment]
Oh, you probably refer to something like this:
```
> fd mod .. -a
/home/shark/Dropbox/Informatik/rust/fd/src/../src/exec/mod.rs
/home/shark/Dropbox/Informatik/rust/fd/src/../src/fshelper/mod.rs
/home/shark/Dropbox/Informatik/rust/fd/src/../src/lscolors/mod.rs
/home/shark/Dropbox/Informatik/rust/fd/src/../tests/testenv/mod.rs
```

This actually seems like a bug. The paths should be normalized.

**Comment by sharkdp** [comment]
Fixed via #270

**Comment by drahnr** [comment]
@sharkdp awesome, that was fast :)

**Comment by sharkdp** [comment]
Released in [fd v7.0.0](https://github.com/sharkdp/fd/releases/tag/v7.0.0)
